config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
 	performance: {hints: false}, //To mute the performance warnings for the production build...
 	entry: {
 		app: ['babel-polyfill', './src/index.js']
-    },
+    },//entry point of project i.e index.js file.
```

Comment:
I think this comment is not needed here because this is very basic thing in webpack and also object name is already telling this is the entry point.